lib/diagnostic.rb

```diff
@@ -7,36 +7,39 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+  list.sort_by(&:length)
```

Comment:
The questions asks you to implement a method that returns a FIXNUM. What does `sort_by` return? Here's the documentation: http://ruby-doc.org/core-2.3.1/Enumerable.html#method-i-sort_by

Here's 1 solution:
```
  def longest(list)
    list.max { |a, b| a.length <=> b.length }
  end
```